sdk/identity/azure-identity/azure/identity/_internal/decorators.py

```diff
@@ -9,6 +9,19 @@
 from azure.core.exceptions import ClientAuthenticationError
 
 
+def at_least_one_scope(get_token):
```

Comment:
By having a decorator, you only change two lines of code into one line of code at the cost of (some) runtime cost (calling additional methods). I would do the parameter check in the method body instead.